aiken-contracts/fractional/scripts/00_createScriptReferences.sh

```diff
@@ -152,7 +152,7 @@ ${cli} transaction build-raw \
     --tx-out-reference-script-file ${cip68_script_path} \
     --fee 900000
 
-FEE=$(cardano-cli transaction calculate-min-fee --tx-body-file ./tmp/tx.draft --testnet-magic ${testnet_magic} --protocol-params-file ./tmp/protocol.json --tx-in-count 0 --tx-out-count 0 --witness-count 1)
+FEE=$(cardano-cli transaction calculate-min-fee --tx-body-file ./tmp/tx.draft --testnet-magic ${testnet_magic} --protocol-params-file ./tmp/protocol.json --tx-in-count 1 --tx-out-count 1 --witness-count 1)
```

Comment:
We'll need to run through all these fee calculations with the next version of the node. George from MLabs has modified the fee calculations so they're actually correct.  I believe the --tx-in-count and --tx-out-count don't do anything anymore. We'll need to re-test once this gets merged and into a release: https://github.com/IntersectMBO/cardano-cli/pull/534

Reply:
I'll keep an eye on the PR. Seems like it should be merged anytime now. We can redo all these fee calculations after that.